2.2/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openshift/base-centos7
+
+# This image provides a Ruby 2.2 environment you can use to run your Ruby
+# applications.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
```

Comment:
I created the patch, but I don't feel to be maintainer :) But this is not show stopper. We can change it any time later ...